src/app/(header-footer)/search/components/SearchFilter.tsx

```
@@ -0,0 +1,126 @@
+    const changeCount = async () => {
+      const response = await getFilterCount(filter);
+
+      if (!response.success) {
+        switch (response.status) {
+          case 500:
+            toast({
+              title: '네트워크 에러',
+              description: response.message,
+              variant: 'destructive',
+            });
+          default:
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)